src/context/mapStateSlice/index.ts

```diff
@@ -48,17 +49,29 @@ export const mapStateSlice = createSlice({
   initialState,
   reducers: {
     addLayer: ({ layers, ...rest }, { payload }: PayloadAction<LayerType>) => {
-      const layersToAdd = payload?.group?.activateAll
+      const updatedPayload = {
+        ...payload,
+        ...(payload.type === 'boundary' &&
+          countryBaseURL && {
+            path: `${countryBaseURL}/${payload.path}`,
+          }),
+      };
```

Comment:
can you explain why we're making this change?

Reply:
We need this to add baseURL to path field if countryBaseURL available